content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -0,0 +1,332 @@
+a commission can be selected from the drop-down menu. This commission is rewarded to the contact
+selected in the :guilabel:`Referrer` field.
+
+Next, if they have not already been auto-populated with the customer's information, enter the
+appropriate addresses in the :guilabel:`Invoice Address` and :guilabel:`Delivery Address` fields.
+Both of these fields are **required**.
+
+Then, if desired, choose a :guilabel:`Quotation Template` from the drop-down field to apply to this
+quotation.
+
+The default date that appears in the :guilabel:`Expiration` field is based on the number configured
+in the :guilabel:`Default Quotation Validity` setting (in :menuselection:`Sales app -->
+Configuration --> Settings`).
+
+.. tip::
+   When using a quotation template, the date in the :guilabel:`Expiration` field is based off the
+   :guilabel:`Quotation Validity` figure on the template form.
+
+If the quotation is for a recurring product or subscription, select the desired :guilabel:`Recurring
+Plan` from that specific drop-down menu.
+
+If desired, select a specific :guilabel:`Pricelist` to be applied to this quotation.
+
+Lastly, select any specific :guilabel:`Payment Terms` to be used for this quotation.
+
+Order Lines tab
+---------------
+
+The first tab on the quotation form is the :guilabel:`Order Lines` tab.
+
+In this tab, select products, and quantities of those products, to add them to the quotation.
+
+There are two ways to add products to the quotation from this tab.
+
+Clicking :guilabel:`Add a product`, selecting the desired item from the :guilabel:`Product`
+drop-down field, and proceeding to adjust the quantity of that selected product, if necessary, is
+one way to add products to a quotation.
+
+Or, click :guilabel:`Catalog` to reveal a separate page, showcasing every item (and every potential
+product variant) in a neatly organized display, with items organizable by :guilabel:`Product
+Category` and :guilabel:`Attributes`.
+
+.. image:: create_quotations/product-catalog.png
+   :align: center
+   :alt: A product catalog accessible via a quotation in the Odoo Sales application.
+
+From here, simply locate the desired items, click the :icon:`fa-shopping-cart` :guilabel:`Add`
+button on the product card(s), and adjust the quantity, if needed. When complete, click the
+:guilabel:`Back to Quotation` button in the upper-left corner to return to the quotation, where the
+newly-selected catalog items can be found in the :guilabel:`Order Lines` tab.
+
+If multiple items should be presented in a more organized way on the quotation, click :guilabel:`Add
+a section`, enter a name for the section, and drag-and-drop that section heading in the desired
+location amongst the items in the :guilabel:`Order Lines` tab. The section heading appears in bold.
+
+If needed, click :guilabel:`Add a note` beneath a certain product line, to add a custom note about
+that specific product. The note appears in italics.
```

Comment:
Looks like the note can only be at the bottom- so it doesn't appear under whatever product the note is needed for.  So if you need to add a note to a product, you have to add it, make sure it's at the bottom- add the note, then add more products, etc. I tested this a few times and it seems to be how it works. I'd add an "important' admonition or something similar since this could e a huge issue if someone adds a lot of products, thinking they can add notes after they are added.

Reply:
Well, actually, when you write a note, it appears as a line on the Order Lines tab (just like a product or section). And, just like a product or section line, they can be moved and placed anywhere on the quotation. I added some more context about that functionality. 👍